src/api/operations/client.queries.graphql

```diff
@@ -210,6 +210,15 @@ query GetEnrollment($id: ID!) {
   }
 }
 
+query GetEnrollmentReminders($id: ID!) {
```

Comment:
This can go in src/api/operations/enrollment.queries.graphql now. Looks like there are a few merge conflicts from the enrollment dash work, sorry about that!